### PR TITLE
Explicitly state that users can label k8s services for FDS if UDS is …

### PR DESCRIPTION
**Pull request by kdorosh**
…enabled

### CODE REVIEW

File: docs/content/advanced_configuration/fds_mode.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ E.g. with
 ```bash
 kubectl label namespace default discovery.solo.io/function_discovery=disabled
 kubectl label upstream -n myapp myupstream discovery.solo.io/function_discovery=disabled
+
+# if the Upstream was discovered and is managed by UDS (Upstream Discovery Service)
+# then you can add the label to the service and it will propagate to the Upstream
+kubectl label service -n myapp myservice discovery.solo.io/function_discovery=disabled
 ```
```
Comment (ilackarms) [inline docs/content/advanced_configuration/fds_mode.md]: we can also show the equivalent command for `upstream` here

Comment (kdorosh) [reply docs/content/advanced_configuration/fds_mode.md]: we do that right above, do we not?

File: docs/content/advanced_configuration/fds_mode.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ weight: 10
...
 ---
 
-Gloo's **Function Discovery Service** (FDS) attempts to poll service endpoints for:
+Gloo's **Function Discovery Service** (FDS) attempts to poll endpoints for:
 
 * A path serving a [Swagger Document](https://swagger.io/specification/).
 * gRPC Services with [gRPC Reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) enabled.
 
-This means that the Gloo `discovery` pod/binary will make HTTP requests to all services known to Gloo. 
+This means that the Gloo `discovery` pod will make HTTP requests to all services known to Gloo (i.e., defined in `Upstreams`).
```
Comment (ilackarms) [inline docs/content/advanced_configuration/fds_mode.md]: How about:

```
This means that the Gloo `discovery` pod will make HTTP requests to all *Upstreams* known to Gloo. Upstreams can be created automatically by `discovery` from Kubernetes & Consul services, or created manually by users. 
```

Comment (kdorosh) [reply docs/content/advanced_configuration/fds_mode.md]: given that this is under "advanced configuration" i think explaining how to create/manage upstreams is bloat/unnecessary

File: docs/content/advanced_configuration/fds_mode.md (hunk 1 above)
Comment (ilackarms) [reply docs/content/advanced_configuration/fds_mode.md]: 👍

File: docs/content/advanced_configuration/fds_mode.md (hunk 2 above)
Comment (ilackarms) [reply docs/content/advanced_configuration/fds_mode.md]: `all services known to Gloo` is a bit overloaded here. I want to be clear that Gloo connects to Upstreams (which may be backed by kube services, depending on the upstream type)

Comment (kdorosh) [reply docs/content/advanced_configuration/fds_mode.md]: https://github.com/solo-io/gloo/pull/2411/commits/c608b59442e6746c5c968bf023540ca6fa78dc0c

**Pull request merged**